### PR TITLE
Address June 7 bug bash feedback

### PR DESCRIPTION
**Pull request by kemmerle**
## Description and Context
I am addressing some feedback from the June 7 bug bash: 

1) Revise some of the copy in the warning we display to users before they can migrate their app.
2) Revise the error warning to match mocks when users run the command from a non-developer account. 

There was other feedback in the Bug Bash Google Sheet, for which I've left detailed notes. 

## Screenshots

Revised warning copy and new warnings when user attempts to abort command:

Screenshot 2024-06-11 at 11 37 50 AM

Revised error message for non-developer accounts:

Screenshot 2024-06-10 at 2 26 03 PM

## TODO

- [x] Add message when user attempts to abort command. 
- [x] Address any feedback

## Who to Notify

@markhazlewood @brandenrodgers @camden11 @joe-yeager

### CODE REVIEW

File: packages/cli/commands/project/migrateApp.js [hunk 1]
```diff
@@ -121,6 +125,7 @@ exports.handler = async options => {
     type: 'confirm',
     message: i18n(`${i18nKey}.createAppPrompt`),
   });
+  process.stdin.resume();
```
Comment (kemmerle) [inline packages/cli/commands/project/migrateApp.js:128]: See https://github.com/SBoudrias/Inquirer.js/issues/662

File: packages/cli/lang/en.lyaml [hunk 2]
```diff
@@ -514,18 +514,18 @@ en:
...
-            projectConversion: "{{#bold}}The selected app will be permanently converted to a project component.{{/bold}}"
+            title: "{{#bold}}Migrate {{appName}} to the projects framework?{{/bold}}"
+            projectConversion: "{{#bold}}The selected app will be converted to a project component.{{/bold}}"
             appConfig: "All supported app configuration will be moved to the {{#bold}}public-app.json{{/bold}} component definition file. Future updates to those features must be made through the project build and deploy pipeline, not the developer account UI."
             buildAndDeploy: "This will create a new project with a single app component and immediately build and deploy it to your developer account (build #1)."
             existingApps: "{{#bold}}This will not affect existing app users or installs.{{/bold}}"
             copyApp: "We strongly recommend making a copy of your app to test this process in a development app before replacing production."
+          migrationInterrupted: "\nThe command is terminated, but app migration is still in progress. Please check your account to ensure that the project and associated app have been created successfully."
```
Comment (kemmerle) [inline packages/cli/lang/en.lyaml:523]: Still waiting on feedback from @markhazlewood for final copy.

Comment (kemmerle) [reply packages/cli/lang/en.lyaml:523]: Done!

File: packages/cli/commands/project/migrateApp.js [hunk 3]
```diff
@@ -133,10 +138,19 @@ exports.handler = async options => {
       text: i18n(`${i18nKey}.migrationStatus.inProgress`),
     });
 
+    handleKeypress(async key => {
+      if ((key.ctrl && key.name === 'c') || key.name === 'q') {
+        SpinniesManager.remove('migrateApp');
+        logger.log(i18n(`${i18nKey}.migrationInterrupted`));
+        process.exit();
```
Comment (brandenrodgers) [inline packages/cli/commands/project/migrateApp.js:145]: Should we use `EXIT_CODES.SUCCESS` here?

File: packages/cli/commands/project/migrateApp.js (hunk 1 above)
Comment (brandenrodgers) [reply packages/cli/commands/project/migrateApp.js:128]: Nice, good find. Does this work without the `process.stdin.setRawMode(true);` part? I saw one person mention needing that.

**Review by brandenrodgers — APPROVED** [review]
👌

File: packages/cli/commands/project/migrateApp.js (hunk 1 above)
Comment (kemmerle) [reply packages/cli/commands/project/migrateApp.js:128]: It does work without `process.stdin.setRawMode(true);`.

File: packages/cli/commands/project/migrateApp.js (hunk 3 above)
Comment (kemmerle) [reply packages/cli/commands/project/migrateApp.js]: Addressed in https://github.com/HubSpot/hubspot-cli/pull/1095/commits/26c2d72b52322f762d78188eadc2184cdb5ed7d2

**Pull request merged**